mobot/mobot_client/payments.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) 2021 MobileCoin. All rights reserved.
+
+import full_service_cli as mc
+import time
+
+from mobot_client.drop_session import ItemSessionState
+from mobot_client.models import (
+    Order,
+    Sku,
+)
+from decimal import Decimal
+
+from mobot_client.chat_strings import ChatStrings
+
+
+class Payments:
+    """The Payments class handles the logic relevant to sending MOB and handling receipts."""
+
+    def __init__(
+            self, mobilecoin_client, minimum_fee_pmob, account_id, store, messenger, signal
+    ):
+        self.mcc = mobilecoin_client
+        self.minimum_fee_pmob = minimum_fee_pmob
+        self.account_id = account_id
+        self.store = (store,)
```

Comment:
This is a bug (should be just `store` as this is a tuple), which indicates https://github.com/mobilecoinofficial/mobot/blob/2f369f32ec84b2882bff0387c20811fe9b3c20d4/mobot/mobot_client/payments.py#L46 has not been tested recently